network-api/networkapi/wagtailpages/migrations/0094_make_petition_id_required.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.20 on 2023-08-01 21:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("wagtailpages", "0093_auto_20230726_2248"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="petition",
+            name="campaign_id",
+            field=models.CharField(
+                help_text="Which Salesforce campaign identifier should this petition be tied to?",
```

Comment:
```suggestion
                help_text="Which Salesforce Campaign ID should this petition be tied to?",
```

Per new request on dev Slack channel